### PR TITLE
Bump networking for dcos enterprise compatbility

### PR DESCRIPTION
**Pull request by sargun**

### CODE REVIEW

**Comment by sargun** [comment]
@spahl Does this approach look okay?

**Comment by sargun** [comment]
We would want to use notify unknown on upgrades. But otherwise I think it's reasonable. I want to make sure you think so before integrating.

**Comment by spahl** [comment]
Looking

**Comment by sargun** [comment]
@spahl Nudge? If this approach is reasonable, I'll start integrating dcos-enterprise atop this today.

**Comment by spahl** [comment]
Yes this seems good to me. @alberts

File: packages/bootstrap/extra/dcos_internal_utils/bootstrap/cli.py [hunk 1]
```diff
@@ -84,4 +102,9 @@ def parse_args():
         type=str,
         default='/opt/mesosphere/etc/master_count',
         help='File with number of master servers')
+    parser.add_argument(
```
Comment (alberts) [inline packages/bootstrap/extra/dcos_internal_utils/bootstrap/cli.py]: why is this a thing?

**Pull request closed without merging**